biomage/stage/stage.py

```diff
@@ -206,12 +206,20 @@ def get_sandbox_id(templates, manifests, all_experiments, auto):
 
     # if we are in auto mode (non interactive) just generate a random sandbox ID name
     if auto:
-        return (user_name + "-" + random_name.generate_name(
-            lists=(
-                random_name.dictionaries.ADJECTIVES,
-                random_name.dictionaries.ANIMALS,
+        return (
+            (
+                user_name
+                + "-"
+                + random_name.generate_name(
```

Comment:
Was this done by the linter? I feel like it was more readable without everything so split up